src/core/thread/key_manager.cpp

```diff
@@ -476,40 +478,54 @@ void KeyManager::SetKek(const Kek &aKek)
 
 void KeyManager::SetSecurityPolicy(const SecurityPolicy &aSecurityPolicy)
 {
-    if (aSecurityPolicy.mRotationTime < SecurityPolicy::kMinKeyRotationTime)
+    VerifyOrExit(aSecurityPolicy.mRotationTime > 0, LogNote("Rejecting Key Rotation Time of zero"));
```

Comment:
current SPEC JIRA proposed text has a somewhat simpler solution: if Rotation Time is < 2, use the value `2`.  I like this more as it is less disruptive and keeps the guard time always > 0.

Reply:
Good idea. Changed it in the new pushed commit.